crates/aptos-jwk-consensus/src/network_interface.rs

```diff
@@ -36,15 +37,37 @@ impl<NetworkClient: NetworkClientInterface<JWKConsensusMsg>>
         Self { network_client }
     }
 
-    pub async fn send_rpc(
+    pub async fn send_rpc_raw(
         &self,
         peer: PeerId,
-        message: JWKConsensusMsg,
+        message: Bytes,
         rpc_timeout: Duration,
     ) -> Result<JWKConsensusMsg, Error> {
-        let peer_network_id = PeerNetworkId::new(NetworkId::Validator, peer);
+        let peer_network_id = self.get_peer_network_id_for_peer(peer);
         self.network_client
-            .send_to_peer_rpc(message, rpc_timeout, peer_network_id)
+            .send_to_peer_rpc_raw(message, rpc_timeout, peer_network_id)
             .await
     }
+
+    pub fn to_bytes(
```

Comment:
the name is a bit confusing, maybe "to_bytes_per_protocol"?

Reply:
you mean just for the network_interface or everywhere?